Solution1/BusinessLayer/Extensions/BaseWeatherDTOExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BusinessLayer.DTOs;
+
+namespace BusinessLayer.Extensions
+{
+    public static class BaseWeatherDTOExtensions
+    {
+        public static T FillCommentByTemp <T> (this T weatherDTO)
+            where T : BaseWeatherDTO
+        {
+            weatherDTO.Comment = weatherDTO.Temp switch
+            {
+                _ when weatherDTO.Temp < 0 => "Dress warmly.",
```

Comment:
there is sense to use your enum - but you need enum only for string representation - it's not a thing for which enum was created. so just create separate constants class and use it both in tests and here

Reply:
+

Reply:
+